src/helpers/mathjax.coffee

```diff
@@ -19,13 +21,12 @@ cleanMathArtifacts = ->
   ])
 
 
-typesetMath = (root) ->
-  nodes = root.querySelectorAll('[data-math]:not(.math-rendered)') or []
-  hasMath = root.querySelector('math')
 
-  # Return immediatly if no [data-math] or <math> elements are present
-  # TODO: If the MathJax Queue is not available then MathJax has not loaded yet. Add a load callback to enqueue.
-  return unless window.MathJax?.Hub?.Queue? and (_.any(nodes) or hasMath)
+# Typesets the entire document
+TypesetDocument = ->
```

Comment:
since this is a function (& has side-effects) could you make this `typesetDocument` ?